force-app/main/default/classes/SOQL.cls

```diff
@@ -85,9 +101,22 @@ public with sharing class SOQL {
      * @return A map of Task Ids to Task records.
      * Update the corresponding test class and testGetTasksDueNextWeek method to cover this method
      */
-    //[public | private ] [static] data_type method_name (input parameters) {
+    public static Map<Id, Task> GetTasksDueNextWeek(Task tt1, Task tt2, Task tt3){
```

Comment:
You wont pass the tasks in here, you would want to query for them instead. Can you change this method? Also you probably would not want a method signature like this that takes 3 inputs of tasks. Why would it not be a list?